statistics-dashboard-worker/src/main/java/eu/europeana/statistics/dashboard/worker/execution/HistoricalDataRunner.java

```diff
@@ -1,57 +1,26 @@
 package eu.europeana.statistics.dashboard.worker.execution;
 
 import eu.europeana.statistics.dashboard.common.internal.model.Historical;
-import eu.europeana.statistics.dashboard.common.internal.MongoStatisticsField;
 import eu.europeana.statistics.dashboard.service.persistence.MongoSDDao;
 import eu.europeana.statistics.dashboard.service.persistence.StatisticsData;
 import eu.europeana.statistics.dashboard.service.persistence.StatisticsQuery;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-
-import java.time.LocalDateTime;
 import java.util.List;
 
-
 public class HistoricalDataRunner{
 
     private static final Logger LOGGER = LoggerFactory.getLogger(AnalyzerRunner.class);
```

Comment:
The Logger here is incorrect, it should be
`LoggerFactory.getLogger(HistoricalDataRunner.class)`

Reply:
Fixed